main.pm

```diff
@@ -24,6 +24,7 @@ sub load_install_tests() {
     # interaction
     return 1 if get_var('OPENQA_FROM_GIT');
     loadtest 'install/openqa_worker' unless get_var('OPENQA_FROM_BOOTSTRAP');
+    loadtest 'install/worker_ay_validation'; # unless get_var('VALIDATE_AUTOYAST');
```

Comment:
This should probably be `if` and *not* `unless`.
```suggestion
    loadtest 'install/worker_ay_validation'  if get_var('VALIDATE_AUTOYAST');
```

Reply:
Done. I had forgotten about that